core-bundle/src/Resources/contao/classes/BackendUser.php

```diff
@@ -180,7 +180,7 @@ public function authenticate()
 			$parameters['referer'] = base64_encode($request->getRequestUri());
 		}
 
-		throw new RedirectResponseException(\System::getContainer()->get('router')->generate('contao_backend_login', $parameters, UrlGeneratorInterface::ABSOLUTE_URL));
+		throw new RedirectResponseException(\System::getContainer()->get('router')->generate('contao_backend_login', $parameters, UrlGeneratorInterface::ABSOLUTE_URL), 307);
```

Comment:
Since the default value was changed to `307`, this is not necessary.

Reply:
Removed in d66a47e47a557da17b7633d551b8f8038df78e92.